ggml/src/CMakeLists.txt

```diff
@@ -566,6 +566,17 @@ if (GGML_SYCL)
         list(APPEND GGML_EXTRA_LIBS_PRIVATE DNNL::dnnl)
     endif()
 
+    set(oneCCL_DIR "/opt/intel/oneapi/ccl/latest/lib/cmake/oneCCL")
+    set(MPI_INCLUDE_PATH "/opt/intel/oneapi/mpi/latest/include")
+    set(MPI_LIBRARY_PATH "/opt/intel/oneapi/mpi/latest/lib/")
+    set(ONECCL_INCLUDE_PATH "/opt/intel/oneapi/ccl/latest/include")
+    set(ONECCL_LIBRARY_PATH "/opt/intel/oneapi/ccl/latest/lib/")
+    include_directories(${MPI_INCLUDE_PATH} ${ONECCL_INCLUDE_PATH})
+    find_library(MPI_LIBRARY mpi HINTS ${MPI_LIBRARY_PATH})
+    find_library(ONECCL_LIBRARY ccl HINTS ${ONECCL_LIBRARY_PATH})
+    # find_package(oneCCL REQUIRED)
+    message("-- oneCCL found")
```

Comment:
Add script for not found oneCCL.

oneCCL is not included in oneAPI base toolkit, please print the message to guide user how to install it.